### DISCUSSION TITLE
Extensions

### DISCUSSION THREAD

**Discussion by AVor0n** [wladradchenko/wunjo.wladradchenko.ru#4] [Q&A] [answered]
What extensions are there at the moment?

**Comment by wladradchenko** [comment, marked as answer]
> What extensions are there at the moment?

The complete list of extensions is constantly updated. You can also make extensions yourself. More details at https://github.com/wladradchenko/advanced.wunjo.wladradchenko.ru.

At the moment there is an extension advanced and openai. The first extension allows you to train TTS models, watch the console and switch from CPU to GPU. The second extension allows you to generate images by Dalle-2 and create dialogues through ChatGPT.
